api/openshift/cron-materialized-views.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: cronjob-template
+  annotations:
+    description: 'Template for job that curls your api'
+    tags: 'cronjob'
+parameters:
+  - name: CRON_SCHEDULE
+    description: 'Cron-like schedule expression. Default: Once every day at 10:00 AM UTC or 02:00 AM PST'
```

Comment:
We should probably update this to reflect the every 30 minute config on the line below